### PR TITLE
Show risky experimental settings (fixes #6245)

### LINKED ISSUE #6245: Update super secret experimental settings
We have two experimental settings which require pressing shift nine times. We should update this so that they're always shown, but have a disclaimer for the section that explains that there's an elevated chance that this will cause replaying issues.

### PR DESCRIPTION
**Pull request by jasonLaster**
Screen Shot 2022-04-10 at 4 57 53 PM

### CODE REVIEW

**Comment by bhackett1024** [comment]
These settings should be combined into one, "long recording support".  Controllers and snapshots are backend internal concepts that don't belong in the devtools.  I don't know what the point of marking these as especially risky is, experimental settings are experimental for a reason.

**Comment by bhackett1024** [comment]
Here is a patch I was working on to combine the settings, I think these changes should be folded into this PR.

https://gist.github.com/bhackett1024/67145e65b9605f3b350a45e38c7bb4b3

File: src/ui/actions/session.ts [hunk 1]
```diff
@@ -155,8 +155,8 @@ export function createSession(recordingId: string): UIThunkAction {
       const experimentalSettings: ExperimentalSettings = {
         listenForMetrics: !!window.app.prefs.listenForMetrics,
         disableCache: !!window.app.prefs.disableCache,
-        useMultipleControllers: !!window.app.features.useMultipleControllers,
-        multipleControllerUseSnapshots: !!window.app.features.multipleControllerUseSnapshots,
+        useMultipleControllers: !!window.app.features.tenMinuteReplays,
```
Comment (jaril) [inline src/ui/actions/session.ts:158]: can't this just be `tenMinuteReplays`?

Comment (jaril) [reply src/ui/actions/session.ts:158]: Oh I see — nevermindnevermind

**Review by jaril — APPROVED** [review]

**Comment by jaril** [comment]
Re-ran the tests but this looks like this should be ready to go out @jasonLaster

**Pull request merged**